source/renderer/app/config/walletRestoreConfig.js

```diff
@@ -17,6 +18,11 @@ export const RESTORE_WALLET_STEPS: Array<RestoreWalletStep> = [
   'success',
 ];
 
+export const IMPORT_WALLET_STEPS: Array<ImportWalletStep> = [
+  'walletImportFile',
+  'walletSelectImport',
+];
+
```

Comment:
Can we transform this to an array of objects like `[ WALLET_IMPORT_FILE: 'walletImportFile' ]`?

Reply:
Any benefit of converting it? @tomothespian

Reply:
@DeeJayElly yes. Instead calling `IMPORT_WALLET_STEPS[0]` or `IMPORT_WALLET_STEPS [1]` you will use `IMPORT_WALLET_STEPS . WALLET_IMPORT_FILE ` which is much cleaner and more descriptive. We are also using that syntax on other places

Reply:
Done